spec/peep_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'peep'
+
+describe Peep do
+  describe '.all' do
+    it 'Shows a litst of different peeps' do
+      peep = Peep.all
+      expect(peep).to include "Hello Makers"
+      expect(peep).to include "This weekend challenege is fun"
+    end
+  end
+end
```

Comment:
This is a great test to see if multiple peeps have been displayed; this is a really useful test.